docs/elements.md

```diff
@@ -243,6 +243,70 @@ To compute `max-select`, we use a similar algorithm (note the different default
 
 ---
 
+### `pl-dropdown` element
```

Comment:
pl-dropdown is actually at the bottom of this document, in the deprecated section.

Reply:
Huh, weird, I must've accidentally moved that back up when resolving a merge conflict.